README.md

```diff
@@ -35,6 +35,39 @@ Each platform maintains multiple types of releases:
 * Beta: A version that is nearing full "stable" release.
 * Nightly: Contains the very latest code. These versions are published daily whenever the code changes.
 
+# Contributing
+
+npm package maintenance is automated with [Lerna].
+
+## Installing everything for the first time
+
+    npm install
+
+This will install Lerna, and will “bootstrap” all dependencies; i.e.,
+any common dependencies will be installed in the local `node_modules`,
+and any intra-repository dependencies are resolved.
+
+## Incrementing the version number
```

Comment:
Given I think we want to control version numbers entirely through our existing CI, we don't want to do any increments. Instead we want something like `npm run set-version $VERSION` where `$VERSION` is controlled by the CI, and may be `12.0.11-beta` or `12.0.15` or that kinda thing.

Reply:
Time to figure out how to do that... what was the environment variable that's set to do this again?